lib/codegen/ProofEvent.cpp

```diff
@@ -288,7 +288,7 @@ llvm::BasicBlock *proof_event::rewrite_event_pre(
 llvm::BasicBlock *proof_event::rewrite_event_post(
     kore_axiom_declaration *axiom, llvm::Value *return_value,
     llvm::BasicBlock *current_block) {
-  if (!proof_hint_instrumentation) {
+  if (!proof_hint_instrumentation_slow) {
```

Comment:
Is this the only place we need to modify this flag?

Reply:
yes we want to emit the intermediate configs if the slow flag has been passed. If the normal flag has been passed or if no flag has been passed (in both of those cases (`proof_hint_instrumentation_slow` would be `false`) we want to skip this part.